translations.yml

```diff
@@ -970,6 +975,7 @@
   nl: 'in tekst'
   fa: 'در متن'
   id: 'dalam teks'
+  ko: '텍스트에서'
```

Comment:
Yes :-) It is used with links from the end of a footnote back up to the main text on the page where the footnote is indicated.

See here https://www.w3.org/WAI/business-case/#fn:1
 "back to footnote 1 in text"

Does that make sense?

Reply:
Is this ready to merge and publish! :-)

Reply:
I'm going to go ahead and merge this, because I have others to add and I don't want to get out of synch!

Feel free to submit another pull request if you want to change it later.